AUTHORS

```diff
@@ -0,0 +1,195 @@
+# Authors ordered by first contribution.
+
+## (generated with `git log --encoding=utf-8 --full-history --reverse "--format=format:%at;%an;%ae" | sort -t ';' -u -k 3,3 | sort | awk -F ';' '{ print $2 " <" $3 ">" }'`)
```

Comment:
If it's not needed I'll definitely remove it; imo anything that's in version control doesn't belong in files.